qa/rpc-tests/checkblockatheight.py

```diff
@@ -267,18 +272,18 @@ def run_test(self):
         self.setup_network(False, 0)
 
         chunks = 305
-        self.mark_logs("  Node0 generates %d blocks" % chunks)
-        blocks.extend(self.nodes[0].generate(chunks)) 
+        self.mark_logs(f"  Node {MAIN_NODE} generates {chunks} blocks")
+        blocks.extend(self.nodes[MAIN_NODE].generate(chunks)) 
         self.sync_all()
 
-        self.mark_logs("  Split network: (0)---(1)---(2)   (3)")
+        self.mark_logs(f"The network is split: {HONEST_NODES[0]}-{HONEST_NODES[1]}-{HONEST_NODES[2]} .. {DISHONEST_NODE_INDEX}")
         self.split_network()
 
-        self.mark_logs("  Node0 generating 1 honest block")
-        blocks.extend(self.nodes[0].generate(1)) 
+        self.mark_logs(f"  Node {MAIN_NODE} generating 1 honest block")
+        blocks.extend(self.nodes[MAIN_NODE].generate(1)) 
         sync_blocks(self.nodes, 1, False, 5)
 
-        h_current = self.nodes[1].getblockcount()
+        h_current = self.nodes[HONEST_NODES[1]].getblockcount()
 
         # we will perform on attack aimed at reverting from this block (latest generated) upward
         h_attacked = h_current
```

Comment:
Let's take this chance to get rid of the `;` at line 290

Reply:
Well spotted! Also took a chance to remove another sneaky one from `sc_cert_ceasing_split.py`